airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/cdc/MssqlDebeziumStateUtil.java

```diff
@@ -62,7 +62,7 @@ public class MssqlDebeziumStateUtil implements DebeziumStateUtil {
                                                           Set @res = 1
                                                       ELSE
                                                           Set @res = 0
-                                                      select @res as [included], @MIN_LSN as [min], @MAX_LSN as [max]
+                                                      select @res as [included], @min_lsn as [min], @max_lsn as [max]
```

Comment:
not blocking but I think we might need a test for this

Reply:
I would rather simplify this into a `SELECT MIN(start_lsn), max(start_lsn) FROM cdc_change_tables;` and do the business logic in java, if possible